Sources/Passes/PassesError.swift

```diff
@@ -10,10 +10,8 @@ public struct PassesError: Error, Sendable, Equatable {
     /// The type of the errors that can be thrown by PassKit passes.
     public struct ErrorType: Sendable, Hashable, CustomStringConvertible, Equatable {
         enum Base: String, Sendable, Equatable {
-            case templateNotDirectory
-            case pemCertificateMissing
-            case pemPrivateKeyMissing
-            case opensslBinaryMissing
+            case noSourceFiles
```

Comment:
A lot of this seems duplicated. Could we extract it all to a shared module?

Reply:
Yes 👍🏻